charts/tileserver/templates/deployment.yaml

```
@@ -0,0 +1,93 @@
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "tileserver.fullname" . }}
+  labels:
+    {{- include "tileserver.labels" . | nindent 4 }}
+    {{- if $.Values.api.labels }}
+    {{- toYaml $.Values.api.labels | nindent 4 }}
+    {{- end }}
+
+  {{- if $.Values.api.annotations }}
+  annotations:
+    {{- toYaml $.Values.api.annotations | nindent 4 }}
+  {{- end }}
+
+spec:
+  replicas: {{ .Values.api.replicaCount }}
+  revisionHistoryLimit: {{ .Values.api.revisionHistory }}
+
+  selector:
+    matchLabels:
+      {{- include "tileserver.selectorLabels" . | nindent 6 }}
+
+  {{- with .Values.api.strategy }}
+  strategy:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+
+  template:
+    metadata:
```

Стоит добавить `podAnnotations` пустой, тут могут захотеть поставить аннотации прямо на поды, например для prometheus'а.

```suggestion
    metadata:
      {{- with .Values.podAnnotations }}
      annotations:
        {{- toYaml . | nindent 4 }}
      {{- end }}
```

---

сделал